### DISCUSSION TITLE
Revised release date for onnxruntime 1.17

### DISCUSSION THREAD

**Discussion by tdeboissiere** [microsoft/onnxruntime#19176] [Release Q&A]
Hello !

Do you have a revised date for the release of onnxruntime 1.17 ?
The docs indicate early December 2023.

Thank you !

**Comment by fvdnabee** [comment]
The [rel-1.17.0](https://github.com/microsoft/onnxruntime/tree/rel-1.17.0) branch exists, so I'm assuming the release is imminent.

**Comment by Craigacp** [comment]
ORT 1.17.0 RCs are available - https://github.com/microsoft/onnxruntime/issues/19236
